specifications/common/introduction.md

```diff
@@ -1,32 +1,11 @@
 # Introduction
 
-The __Dataspace Protocol__ is used in the context of [=Dataspaces=] as described and defined in the subsequent sections
-with the purpose to support interoperability.
-In this context, the specification provides fundamental technical interoperability for [=Participants=]
-in [=Dataspaces=].
-Beyond the technical interoperability measures described in this specification, semantic interoperability should also be
-addressed by the [=Participants=]. On the perspective of the [=Dataspace=], interoperability needs to be addressed also
-on the level of trust, on organizational levels, and on legal levels.
-The aspect of cross-dataspace communication is not subject of this document, as this is addressed by the [=Dataspaces=]'
-organizational and legal agreements.
+The __Dataspace Protocol__ is used in the context of [=Dataspaces=] as described and defined in the subsequent sections with the purpose to support _interoperability_. In this context, the specification provides fundamental technical interoperability for [=Participants=] in [=Dataspaces=]. Beyond the technical interoperability measures described in this specification, semantic interoperability should be addressed by the [=Participants=]. On the perspective of the [=Dataspace=], interoperability needs to be addressed also on the level of trust, on organizational levels, and on legal levels. The aspect of cross-dataspace communication is not subject of this document, as this is addressed by the [=Dataspaces=]' organizational and legal agreements.
```

Comment:
This may be straying from the  purpose of this PR but I think these types of statements should be in the best practices guide because they deal with "SHOULDs" but not "MUSTs"

Reply:
This is the introduction which shall provide also some context to the document. I would leave it here, but agree that in the main body of the document we should consider the MUSTs and leave the SHOULDs, in general, to the best practices

Reply:
My first thought was to move this to "scope". Then I read the ISO rules again and recognized that the current scope should also not be part of the scope as this must be _normative_. The more I think about this, the more I agree to remove this phrase and also shorten the scope section drastically. One option would be to add these "hints" for the users not only to the best practices document, but also to the `README` of this repo. This way, it is easy to find but not part of the spec files. WDYT?